### PR TITLE
feat(RTE): allow document paths

### PR DESCRIPTION
**Pull request by fulopdaniel**
1. Prepend URL with https:// if not exists.
2. Allow adding URLs that begin with /document/....

### CODE REVIEW

**Review by Walkeezy — APPROVED** [review]
Nice!

File: packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/index.ts [hunk 1]
```diff
@@ -7,8 +7,11 @@ import { CustomFloatingLink } from './FloatingLink/CustomFloatingLink';
 import { LINK_PLUGIN } from './id';
 import { LinkButton } from './LinkButton';
 import { LinkMarkupElement } from './LinkMarkupElement';
+import { relativeUrlRegex, telOrMailRegex, urlRegexWithHttps } from './utils';
 
-export const isValidUrl = (url: string): boolean => !!url.match(/^((https?:\/\/|mailto:|tel:).+)|\/r\/.+/);
+export const isValidUrl = (url: string): boolean => {
```
Comment (ragi96) [inline packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/index.ts:12]: Also that function is worth to have some unit tests

Comment (fulopdaniel) [reply packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/index.ts]: Thanks, done!

File: packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/utils/regex.ts
[the remark this answers is not in the document]
Comment (fulopdaniel) [reply packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/utils/regex.ts:1]: Good idea, I even caught a bug when implementing tests. Done!

**Pull request merged**